build-config/src/main/resources/build-config/checkstyle.xml

```diff
@@ -122,7 +122,7 @@
         </module>
         <module name="MethodLength"/>
         <module name="ParameterNumber">
-            <property name="max" value="10"/>
```

Comment:
We prefer to do this in the source code. For this purpose we have an annotation: `@SuppressWarnings("checkstyle:parameter number")`. See for instance `org.mapstruct.ap.internal.model.source.SourceMethod`